api/controllers/PokemonController.js

```diff
@@ -73,13 +71,7 @@ module.exports = _.mapValues({
 
   async delete (req, res) {
     const id = req.param('id');
-    let pokemon = await Pokemon.findOne({id});
-    if (!pokemon) {
-      return res.notFound();
-    }
-    if (pokemon.owner !== req.user.name && !req.user.isAdmin) {
-      return res.forbidden();
-    }
+    let pokemon = await Validation.verifyUserIsPokemonOwner({user: req.user, id});
```

Comment:
Can we stick to the validation doing just validation, please? I'd prefer if it didn't do the getting of things from the database as well.